### PR TITLE
Embeded dummy tx

### PR DESCRIPTION
**Pull request by mohanson**
Let `--simple-binary` be an alias to `--replace-binary` in order to allow `--listen`, `--pprof` and other parameters.

```sh
# Examples:
$ ckb-debugger --simple-binary fib
# or
$ ckb-debugger --simple-binary fib --pprof /tmp/out
# or
$ ckb-debugger --simple-binary fib --listen 127.0.0.1:2000
```

### CODE REVIEW

File: bins/src/main.rs [hunk 1]
```diff
@@ -145,39 +145,28 @@ fn main() {
         )
         .get_matches();
```
Comment (XuJiandong) [inline bins/src/main.rs:147]: fix this issue:
https://github.com/nervosnetwork/ckb-standalone-debugger/issues/24

**Comment by XuJiandong** [comment]
When following
https://github.com/XuJiandong/ckb-debugger-playground to run simple binary with panic:
```
make run-simple-binary-panic
```
There is no backtrace:
```
ckb-debugger --simple-binary build/panic
Run result: Err(OutOfBound)
Total cycles consumed: 1790
Transfer cycles: 1758, running cycles: 32

Machine status: pc  : 0x           11B20
zero: 0x               0 ra  : 0x           11AF4 sp  : 0x          3FFFD0 gp  : 0x           13368
tp  : 0x               0 t0  : 0x               0 t1  : 0x               0 t2  : 0x               0
s0  : 0x          3FFFF0 s1  : 0x               0 a0  : 0x               0 a1  : 0x          3FFFF8
a2  : 0x               0 a3  : 0x               0 a4  : 0x              64 a5  : 0x    FFFFFFFFFFFF
a6  : 0x               0 a7  : 0x               0 s2  : 0x               0 s3  : 0x               0
s4  : 0x               0 s5  : 0x               0 s6  : 0x               0 s7  : 0x               0
s8  : 0x               0 s9  : 0x               0 s10 : 0x               0 s11 : 0x               0
t3  : 0x               0 t4  : 0x               0 t5  : 0x               0 t6  : 0x               0
```

**Comment by XuJiandong** [comment]
continues:
```
❯ make run-simple-binary-pprof
ckb-debugger --simple-binary build/fib --pprof=build/fib.pprof
thread 'main' panicked at 'attempt to subtract with overflow', src/main.rs:398:25
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
make: *** [run-simple-binary-pprof] Error 101
```

**Comment by mohanson** [comment]
> When following
> https://github.com/XuJiandong/ckb-debugger-playground to run simple binary with panic:
> 
> ```
> make run-simple-binary-panic
> ```
> 
> There is no backtrace:
> 
> ```
> ckb-debugger --simple-binary build/panic
> Run result: Err(OutOfBound)
> Total cycles consumed: 1790
> Transfer cycles: 1758, running cycles: 32
> 
> Machine status: pc  : 0x           11B20
> zero: 0x               0 ra  : 0x           11AF4 sp  : 0x          3FFFD0 gp  : 0x           13368
> tp  : 0x               0 t0  : 0x               0 t1  : 0x               0 t2  : 0x               0
> s0  : 0x          3FFFF0 s1  : 0x               0 a0  : 0x               0 a1  : 0x          3FFFF8
> a2  : 0x               0 a3  : 0x               0 a4  : 0x              64 a5  : 0x    FFFFFFFFFFFF
> a6  : 0x               0 a7  : 0x               0 s2  : 0x               0 s3  : 0x               0
> s4  : 0x               0 s5  : 0x               0 s6  : 0x               0 s7  : 0x               0
> s8  : 0x               0 s9  : 0x               0 s10 : 0x               0 s11 : 0x               0
> t3  : 0x               0 t4  : 0x               0 t5  : 0x               0 t6  : 0x               0
> ```

backtrace needs `--pprof`

**Comment by mohanson** [comment]
> continues:
> 
> ```
> ❯ make run-simple-binary-pprof
> ckb-debugger --simple-binary build/fib --pprof=build/fib.pprof
> thread 'main' panicked at 'attempt to subtract with overflow', src/main.rs:398:25
> note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
> make: *** [run-simple-binary-pprof] Error 101
> ```

Fix in #26

**Pull request merged**